0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -297,6 +298,11 @@ if transfer is not conditional.
 
 This MAY be used by receivers or connectors to reject incoming funds if they will not fulfill the condition or are unable to forward the payment. Previous hops in an Interledger transfer would have their money returned before the expiry and the sender or previous connectors MAY retry and reroute the transfer through an alternate path.
 
+#### cancelIncomingTransfer
```

Comment:
My intention was for these to be different in the same way `outgoing_cancel` is different from `outgoing_reject`, namely 'cancel' means rejected at the ledger level instead of at the connector level (e.g. timeout, malformed, negative amount, insufficient account balance). But maybe we can group them together. After all, `sendTransfer` also relates to two different events, `incoming_transfer` for unconditional and `incoming_prepare` for conditional.